FNA.Core.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
-		<Platforms>x64</Platforms>
+		<TargetFrameworks>net40;netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
I believe this will break due to the reasons listed here: https://github.com/FNA-XNA/FNA/pull/355#issuecomment-876070586

Reply:
It seems to work fine for me... maybe it was a fluke? SDL2# is set up the same way https://github.com/flibitijibibo/SDL2-CS/blob/master/SDL2-CS.Core.csproj#L3

Reply:
> VS 2022 and OmniSharp are forcing our hand here, since Intellisense has a very bad time when referencing projects that target both Framework and Core.

@flibitijibibo  Is there any way to repro the issue you saw? I'd be curious to follow up the root cause issue of this